setup.cfg

```diff
@@ -65,7 +65,6 @@ install_requires =
     # Invenio-App-RDM
     invenio-rdm-records>=1.1.0,<2.0.0
     CairoSVG>=2.5.2,<3.0.0
-    invenio-pages>=1.0.0a5,<1.1.0
```

Comment:
why is this removed? shouldn't be bumped to the major release of invenio-pages?

Reply:
It was moved to invenio-rdm-records, see [here](https://github.com/inveniosoftware/invenio-rdm-records/pull/1142/files#diff-fa602a8a75dc9dcc92261bac5f533c2a85e34fcceaff63b3a3a81d9acde2fc52R46).